staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers_test.go

```diff
@@ -391,18 +480,24 @@ func benchmarkSerializeObject(b *testing.B, size int) {
 	}
 }
 
-func BenchmarkSerializeObject10KB(b *testing.B) {
-	benchmarkSerializeObject(b, 10*1024)
+func BenchmarkSerializeObject1000PodsPB(b *testing.B) {
+	benchmarkSerializeObject(b, toProtoBuf(b, benchmarkItems(b, "testdata/pod.json", 1000)))
```

Comment:
You may want to reset the time after computing benchmark items, something like:

items := benchmarkItems(...)
b.ResetTimer()
benchmarkSerializeObject(...)

Reply:
Ok - it's done later anyway. Never mind.